packages/portfolio-contract/test/supports.ts

```diff
@@ -17,6 +18,32 @@ export {
   makeFakeTransferBridge,
 } from '@agoric/vats/tools/fake-bridge.js';
 
+// Deep clone fetchedChainInfo for test patching
+const testChainInfo = deepCopyJsonable(fetchedChainInfo);
+// Patch noble.connections to add axelar-dojo-1 for test
+testChainInfo.noble.connections['axelar-dojo-1'] = {
```

Comment:
suggest: use a spread rather than mutation:

```js
const testChainInfo = {
  ...fetchedChainInfo,
  noble: {
    ...fetchedChainInfo.noble,
    connections: {
       ...fetchedChainInfo.noble.connections,
       'axelar-dojo-1': { ...}
}}}
```